.github/workflows/gopy.yml

```diff
@@ -28,16 +28,40 @@ jobs:
       - name: Set up Python
         uses: actions/setup-python@v2
         with:
-          python-version: "3.11"
+          python-version: "3.11.10"
 
       - name: Install pybindgen
         run: |
           python3 -m pip install pybindgen
           go install golang.org/x/tools/cmd/goimports@latest
           go install github.com/go-python/gopy@latest
 
-      - name: Build
+      - name: Build python module
         run: |
           cd src/pkg/client
-          gopy pkg -author="Photoroom" -email="team@photoroom.com" -url="" -name="datago" -version="0.99" .
-          ls datago/*
+          gopy pkg -author="Photoroom" -email="team@photoroom.com" -name="datago" .
+          export DESTINATION="../../../build"
```

Comment:
move the built module around, eventually we'll be able to automate the pypi / twine upload